src/NHibernate.Validator/Engine/ClassValidator.cs

```diff
@@ -687,7 +864,7 @@ private IEnumerable<InvalidValue> MembersInvalidValues(object entity, string mem
 				throw new TargetException(
 					string.Format("The property or field '{0}' was not found in class {1}", memberName, entityType.FullName));
 			}
-		    
+			

```

Comment:
Better remove all the spaces from blank lines, or do not change them for avoiding raising the count of changes to review.

Reply:
Removed. Again it was done by tabify function